float64[] arm_joints
geometry_msgs/Pose pick_pose
geometry_msgs/Pose place_pose
float64[] asm_pose1

